python/tk_houdini/ui_generation.py

```diff
@@ -447,3 +550,74 @@ def _jump_to_fs(engine):
     engine.launch_command('%s')
 """
 
+# The code that is stored in the python panel interfaces. 
+_g_panel_script = \
+"""
+from PySide import QtGui
+
+class NoPanelWidget(QtGui.QWidget):
+    
+    def __init__(self, msg, error=None):
+    
+        super(NoPanelWidget, self).__init__()
+
+        sg_icon_path = '%s'
+        
+        sg_icon = QtGui.QLabel()
+        sg_icon.setPixmap(QtGui.QPixmap(sg_icon_path))
+        msg_lbl = QtGui.QLabel(msg) 
+        msg_lbl.setWordWrap(True)
+
+        if error:
+            error_txt = QtGui.QTextEdit(error)
+            error_txt.setReadOnly(True)
+
+        h_layout = QtGui.QHBoxLayout()
+        h_layout.setSpacing(5)
+        h_layout.addWidget(sg_icon)
+        h_layout.addWidget(msg_lbl)
+    
+        v_layout = QtGui.QVBoxLayout(self)
+        v_layout.setContentsMargins(10, 10, 10, 10)
+        v_layout.setSpacing(15)
+        v_layout.addStretch()
+        v_layout.addLayout(h_layout)
+        if error:
+            v_layout.addWidget(error_txt)
+        v_layout.addStretch()
+
+def createInterface():
+
+    try:
+        import tank.platform.engine
+    except ImportError:    
+        return NoPanelidget(
+            "It looks like you're running Houdini outside of a Shotgun "
```

Comment:
haha! Funny how my comment made you spot that. And sorry for not spotting in CR!